Leanplum-SDK/Classes/Features/Actions/LPActionManager.m

```diff
@@ -752,7 +752,7 @@ - (void)didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)token
     }
 
     // Format push token.
-    NSString *formattedToken = [token description];
+    NSString *formattedToken = [self hexadecimalStringFromData:token];
```

Comment:
Wont this break, for people who havent upgraded to iOS13? Shouldnt this be under a check for OS version.

Reply:
no it will not